salt/utils/etcd_util.py

```diff
@@ -249,6 +261,29 @@ def read(self, key, recursive=False, wait=False, timeout=None, waitIndex=None):
         return result
 
     def _flatten(self, data, path=""):
+        """
+        Take a data dictionary and flatten it to a dictionary with values that are all strings.
+
+        If path is given, prepend it to all keys.
+
+        For example, given path="/salt" it will convert...
+
+        {
+            "key1": "value1",
```

Comment:
**question** what if there's `"key3": 42,` in this dictionary?

Reply:
sorry, the comment is wrong, it should be 
`Take a data dictionary and flatten it to a dictionary with keys that are all strings.`